test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetSourcesTests.cs

```diff
@@ -17,6 +17,8 @@ namespace Dotnet.Integration.Test
     public class DotnetSourcesTests
     {
         private readonly DotnetIntegrationTestFixture _fixture;
+        string _httpErrorSingleShort = "You are running the '{0}' operation with an 'HTTP' source: {1}. NuGet requires HTTPS sources. Please refer to https://aka.ms/nuget-https-everywhere.";
```

Comment:
Might've missed this in other places and previous PRs, but I think you should use the resource not the string to allow the tests to work on other locales.

Reply:
It's because the Nuget.Commands locals(internal) are not accessible from here.

Reply:
I see, can you add an internals visible to?
Maybe just match https://aka.ms/nuget-https-everywhere?
Check exit code?

Reply:
aka.ms link seems to be broken. I found https://devblogs.microsoft.com/nuget/https-everywhere/ incase if it helps.